main.go

```diff
@@ -2,17 +2,18 @@ package main
 
 import (
 	"fmt"
-	"github.com/spf13/viper"
 	"log"
+
+	"github.com/spf13/viper"
 )
 
 type Config struct {
 	Plex struct {
-		Host  string
-		Token string
+		Host  string `yaml:"host"`
+		Token string `yaml:"token"`
 	}
 	API struct {
-		ListenAddr string
+		ListenAddr string `yaml:"listen_addr"`
```

Comment:
I think this is probably due to the fact the field in the yaml is called HTTP instead of API but I thought it was okay to be explicit.